frontends/web/src/components/transactions/transaction.jsx

```diff
@@ -71,108 +55,171 @@ export default class Transaction extends Component {
         weight,
         numConfirmations,
         time,
+        fiatCode,
         addresses,
+        status,
     }, {
         collapsed,
     }) {
         const badge = t(`transaction.badge.${type}`);
-        const arrow = badge === 'In' ? ArrowDown : badge === 'Out' ? ArrowUp : ArrowRight;
         const sign = ((type === 'send') && '−') || ((type === 'receive') && '+') || null;
-        const date = time ? this.parseTime(time) : (numConfirmations <= 0 ? t('transaction.pending') : 'Time not yet available');
-        const sDate = time ? this.parseTimeShort(time) : (numConfirmations <= 0 ? t('transaction.pending') : 'Time not yet available');
+        const sDate = time ? this.parseTimeShort(time) : '---';
+        const statusStyle = {
+            'complete': style.statusIndicatorComplete,
+            'pending': style.statusIndicatorPending,
+            'failed': style.statusIndicatorFailed,
+        }[status];
+        const statusText = {
+            'complete': "Complete",
+            'pending': "Pending",
+            'failed': "Failed",
+        }[status];
         return (
-            <div class={[style.transactionContainer, collapsed ? style.collapsed : style.expanded].join(' ')}>
-                <div class={['flex flex-column flex-start', style.transaction].join(' ')}>
-                    <div class={['flex flex-row flex-between flex-items-start', style.row].join(' ')}>
-                        <div class="flex flex-row flex-start flex-items-start">
-                            <div class={style.labelContainer} onClick={this.onUncollapse}>
-                                <div class={style.toggleContainer}>
-                                    <div class={[style.toggle, style[type], collapsed ? style.collapsed : style.expanded].join(' ')}></div>
-                                </div>
-                                <div class={[style.transactionLabel, style[type], style.flat].join(' ')}>
-                                    <img src={arrow} />
-                                    {badge}
-                                </div>
-                            </div>
-                            <div>
-                                <div class={style.date}>
-                                    <span>{date}</span>
-                                    <span>{sDate}</span>
-                                </div>
-                                <div class={[style.address, style.multiline].join(' ')}>{addresses.join(', ')}</div>
-                            </div>
+            <div className={[style.container, collapsed ? style.collapsed : '', index === 0 ? style.first : ''].join(' ')}>
+                <div className={[parentStyle.columns, style.row].join(' ')}>
+                    <div className={parentStyle.columnGroup}>
+                        <div className={parentStyle.type}>
+                            {
+                                badge === 'In' ? (
```

Comment:
`badge` is a translated text! Use `type` instead (can be `receive`, `send` or `send_to_self`).

Looks like the badges have been broken in other languages than English for a while now :/